srcalc/build.gradle

```diff
@@ -10,6 +10,7 @@ apply plugin: 'jacoco'
 
 // Basic build properties
 sourceCompatibility = '1.7'
+version = '0.5.0-alpha'
```

Comment:
Will the future version number be 0.<sprint number>.0-alpha? What's the convention we are using here.